models/order.js

```diff
@@ -12,7 +13,68 @@ const Entry = new Schema({
 const orderSchema = new Schema({
   id: String,
   status: { type: String, default: constants.OPEN },
-  items: [Entry]
+  entries: [Entry]
 });
 
+orderSchema.statics.getOrders = function() {
+  return getOrders.apply(this)
```

Comment:
Using apply here because i think 'this' is not longer Order when called inside the promise, i didn't double check though.